docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -233,7 +234,10 @@ Configure the job cluster with the following settings:
 
 - In the `name` field enter `kedro_deployment_demo`.
 - Select the radio button for `Single node`.
-- Select the runtime `12.2 LTS` in the `Databricks runtime version` field.
+- Select the runtime `13.3 LTS` in the `Databricks runtime version` field.
+- In the `Advanced options` section, under the `Spark` tab, locate the `Environment variables` field. Add the following line:
+`KEDRO_LOGGING_CONFIG="/dbfs/FileStore/iris-databricks/conf/logging.yml"`
+Here, ensure you specify the correct path to your custom logging configuration. This step is crucial because the default Kedro logging configuration incorporates the rich library, which is incompatible with Databricks jobs. In the `databricks-iris` Kedro starter, the `rich` handler in `logging.yml` is altered to a `console` handler for compatibility. For additional information about logging configurations, refer to the [Kedro Logging Manual](https://docs.kedro.org/en/stable/logging/logging.html).
```

Comment:
What happens if logging is not set? Does it causes a Databricks job to fail or it create malform logs?

Reply:
Job will work well and will be marked as Green, but it just be malform logs